cmd/jaeger/internal/exporters/storageexporter/factory.go

```diff
@@ -29,18 +30,61 @@ func NewFactory() exporter.Factory {
 }
 
 func createDefaultConfig() component.Config {
-	return &Config{}
+	return &Config{
+		RetryConfig: configretry.BackOffConfig{
+			Enabled:             true,
+			InitialInterval:     5 * time.Second,
+			RandomizationFactor: 0.5,
+			Multiplier:          1.5,
+			MaxInterval:         30 * time.Second,
+			MaxElapsedTime:      5 * time.Minute,
+		},
+	}
 }
 
-func createTracesExporter(ctx context.Context, set exporter.Settings, config component.Config) (exporter.Traces, error) {
+func createTracesExporter(
+	ctx context.Context,
+	set exporter.Settings,
+	config component.Config,
+) (exporter.Traces, error) {
 	cfg := config.(*Config)
+
+	defaultCfg := createDefaultConfig().(*Config)
+
+	if !cfg.RetryConfig.Enabled {
```

Comment:
we didn't have anything similar for other configs, why do we need it for this one?